### DISCUSSION TITLE
Native android app proof of concept

### DISCUSSION THREAD

**Discussion by a-na-nas** [Kovah/LinkAce#804] [General]
Hello!
I've been working on an android app using the LinkAce API as a personal project and I think I've reached a stage where it's somewhat usable already and maybe would be worth sharing with the LinkAce community.

The app still lacks major features, so far allowing only for browsing the links by tags or lists and adding new ones from a somewhat intuitive native interface.
I've made the source code available on my Gitlab for now (might consider cloning it over to Github later): [Hyperlink Knapsack](https://gitlab.com/ananasowe/hyperlinks-knapsack).

I was thinking about making a pull request to the documentation page about [third-party LinkAce tools](https://www.linkace.org/docs/v1/general/third-party-tools/), but I'm worried the app might not quite be on the stage where it could be considered useful yet. 
I would love to hear your opinions!

Here is a screenshot of the home screen, so the post isn't just all boring text:
<img src="https://github.com/Kovah/LinkAce/assets/11874643/de359b27-ba5f-4b14-9a67-ec3b82bf65b7" height="450px"/>

**Comment by Kovah** [comment]
Damn this looks cool! Thank you soo much for taking the time to create the app for LinkAce! 😍 
I would add the app to the documentation, as more interested users might be a good push for it. (You can also do the PR if you want)

**Reply by a-na-nas** [reply]
Thank you! Do you think it would it be best to add it in between the CLI client and browser extensions, or at the end?

**Reply by Kovah** [reply]
Feel free to add a new category for mobile apps at the top. :)

**Comment by Pozi09** [comment]
Hi! Me and my team help aspiring developers with their apps , email me we have lots of cool features
